R/translator.R

```diff
@@ -116,21 +136,44 @@ translator_list <- function() {
 }
 
 
-name_from_context <- function(name = NULL) {
+name_from_context <- function(name = NULL, package = NULL, strict = FALSE) {
```

Comment:
It looks like name here is never used except for in a failure case if passed with `name = "package:something"`. Is that intended? Looks like this means there can only be one translator object for a package.

Reply:
Correct - `named` exist only for the non-package use-case (e.g., use in a shiny app)